src/styles/filters-history.scss

```diff
@@ -117,40 +135,43 @@
 		&:hover {
 			background-color: rgba(0, 0, 0, 0.1);
 		}
+
+		&:active {
+			background-color: rgba(0, 0, 0, 0.3);
+		}
 	}
-	&__pin-icon {
-		& i {
-			display: inline-block;
+	&__filter-pin-icon {
+		margin-bottom: 1px;
+		padding-left: 3px;
+		&:hover {
+			background-color: rgba(0, 0, 0, 0.05);
+		}
+		.pin-icon {
+			color: var(--text-secondary);
+			width: 14px;
+			height: 14px;
 			transition: transform 0.15s;
-			@include icon(url(../../resources/icons/pin.svg), 14px, 14px);
 		}
 		&.pinned {
 			background-color: rgba(0, 0, 0, 0.1);
-			& i {
+			.pin-icon {
 				transform: rotate(-45deg);
 			}
 		}
 	}
-	&__share-icon {
-		position: relative;
-		& i {
-			position: absolute;
-			top: 50%;
-			left: 50%;
-			transform: translate(-50%, -50%);
-			opacity: 0.5;
-			@include icon(url(../../resources/icons/share-option.svg), 14px, 14px);
-		}
+	&__filter-share-icon {
+		color: var(--text-secondary);
+		opacity: 0.5;
 	}
 
 	
-	&__delete-icon {
-		position: relative;
-		& i {
-			position: absolute;
-			transform: translate(-50%, -50%);
-			opacity: 0.5;
-			@include icon(url(../../resources/icons/cross-icon-dark.svg), 16px, 16px);
-		}
+	&__filter-delete-icon {
+		padding-right: 3px;
```

Comment:
if you need different paddings and margins, you can do it like padding: 1px 1px 1px 1px, and I also think that your approach is a bit difficult, try to use margin on parent __controls div